css/index.css

```
@@ -818,3 +818,482 @@ body {
+  gap: 2px;
+  border-radius: 12px;
+}
+
+.most-popular__card{
+  position: relative;
+  overflow: hidden;
+  background-size: cover;
+  background-position: center center;
+  background-color: white;
+  padding: 25px; 
+}
+
+
+.most-popular__link:hover ~ .most-popular__text .most-popular__name {
+  color: rgb(114, 164, 233);
+}
+
+
+.most-popular__link:hover ~ .most-popular__text .most-popular__prices{
+  color: #6c1e1e;
+}
+
+.most-popular__link {
+  font-size: 0;
+  inset: 0;
+  position: absolute;
+  z-index: 1;
+}
+
+.most-popular__badge{ 
+  color: #FFFFFF;
+  background: #05B999;
+  padding: 2px 5px;
+  border-radius: 0px 0px 4px 0px;
+  position: absolute;
+  font-size: 14.11px;
+  font-weight: 600;
+  line-height: 14.11px;
+}
+
+.most-popular__badge--red {
+  background: #EC4C4C;
+  border-radius: 4px;
+}
+
+.most-popular__img {
+  width: 100%; 
+  height: auto; 
+  display: flex;
+  justify-content: center;
+}
+
+.most-popular__text {
+  height: auto; 
+}
+
+.most-popular__name {
+  font-size: 18px; 
+  font-weight: 400;
+  line-height: 22px;
+  text-align: left;
+  color: #716C6C;
+  margin-right: 70px;
+}
+
+.most-popular__prices {
+  font-size: 18px; 
+  font-weight: 700;
+  line-height: 22px;
+  text-align: left;
+  color: #D14D41;
+}
+
+.most-popular__prices--black {
+  font-size: 16px; 
+  color: #292727;
+}
+
+.most-popular__prices--black-2 {
+  font-size: 18px; 
+  color: #292727;
+}
+
+.most-popular__total {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-top: 20px;
+}
+
+
+.most-popular__number {
+  font-size: 12px; 
+  font-style: italic;
+  font-weight: 400;
+  line-height: 16px;
+  text-align: left;
+  color: #A6A6A6;
+}
+
+.most-popular__calification {
+  align-items: center;
+}
+
+.most-popular__star {
+  color: #FDBF00;
+  width: 18px;
+  height: 22px;
+}
+
+.most-popular__rating {
+  font-size: 14px; 
+  font-weight: 400;
+  line-height: 18px;
+  color: #6C6B6A;
+}
+
+
+
+/* Desktop style left*/
+@media (min-width: 768px) {
+  .most-popular {
+    max-width: 1350px;
+    width: 100%;
+    margin: auto;
+    padding: 0px;
+  }
+
+  .most-popular__title {
+    font-size: 24px;
+    font-weight: 700;
+    line-height: 28px;
+    letter-spacing: -1%;
+    color: #363535;
+    margin-top: 51px;
+    margin-bottom: 20px;
+  }
+
+  .most-popular__products {
+    flex-direction: row; 
+  }
+
+  .most-popular__left {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr); 
+    gap: 0px;
+    flex-basis: 773px;
+    border-radius: 12px;
+    background-color: #FFFFFF;
+    padding: 25px 30px; 
+  }
+
+  .most-popular__card {
+    position: relative;
+    overflow: hidden;
+    background-size: cover;
+    background-position: center center;
+    background-color: white;
+    border-radius: 0px;
+    border-bottom: 1px solid #D7D6D6;
+    border-right: 1px solid #D7D6D6;
+    padding: 0px 20px 20px 0px;
+  }
+    
+  .most-popular__card--card-2 {
+    padding: 0px 20px 20px 20px;
+}
```

Las clases de las card tienen este detalle en la tabulacion, que hace pensar que cerraste el media query en vez de la clase
```suggestion
  }
```